docs/md/workload.md

```diff
@@ -0,0 +1,47 @@
+# Workloads
+
+## What is it?
+Within the context of the workflow launcher, a workload is a discrete body of work, which takes data from a source, pushes it into a workflow executor for analysis, and then delivers the results of the analysis to an output location (also known as a sink).
```

Comment:
We generally call it "workflow launcher", not "the workflow launcher"!

Reply:
This sounds like "the google" or "the YouTubes"

Reply:
Sure I can change that. It makes my skin crawl though, because it reads like just bad english :)